llm_toolkit/prompt_builder.py

```diff
@@ -39,6 +39,9 @@
     [FDP_EXAMPLE_2_PROBLEM, FDP_EXAMPLE_2_SOLUTION],
 ]
 
+BUILD_ERROR_SUMMARY = 'The code has the following build issues:'
+FUZZ_ERROR_SUMMARY = 'The code can be built successfully but %s.'
```

Comment:
`FUZZ_ERROR_SUMMARY = 'The code can build successfully but has a runtime issue: '`